src/Card/index.jsx

```diff
@@ -1,12 +1,49 @@
-import Card from 'react-bootstrap/Card';
+import React from 'react';
+import BaseCard from 'react-bootstrap/Card';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import CardContext, { CardContextProvider } from './CardContext';
 import CardHeader from './CardHeader';
+import CardDivider from './CardDivider';
+import CardSection from './CardSection';
+import CardFooter from './CardFooter';
+import CardImageCap from './CardImageCap';
+import CardBody from './CardBody';
+
+const Card = ({ orientation, className, ...props }) => (
```

Comment:
Should this have a `React.forwardRef` on it?

Reply:
It definitely should, added it, thank you!